### PR TITLE
Update .env.example

### PR DESCRIPTION
**Pull request by pyr0gan**
- Tezos env. variable name fixed

### CODE REVIEW

**Comment by guillaumegaluz** [comment]
@pyr0gan you can add me to those reviews, more chances that someone is awake ;) i approved it, feel free to merge

**Review by guillaumegaluz — APPROVED** [review]

**Review by zurgl — APPROVED** [review]
LGTM

**Review by zurgl — CHANGES_REQUESTED** [review]
Rogan do not rename variable of the UI. Thanks you

**Comment by pyr0gan** [comment]
Ack! I didn't mean to push the Celo & Tezos change on this branch! 😢

**Comment by pyr0gan** [comment]
> Rogan do not rename variable of the UI. Thanks you

The pathways are broken without this change. I encountered the issue when making the screen captures. You can test it to make sure 😄

File: components/protocols/celo/components/steps/Swap.tsx [hunk 1]
```diff
@@ -27,9 +27,9 @@ const Change = () => {
 		axios
 			.post(`/api/celo/swap`, state)
 			.then(res => {
-                const hash = res.data.hash
+                const txHash = res.data.txHash
```
Comment (zurgl) [inline components/protocols/celo/components/steps/Swap.tsx:30]: doing this will broke the UI.

**Review by zurgl — CHANGES_REQUESTED** [review]

**Comment by zurgl** [comment]
> > Rogan do not rename variable of the UI. Thanks you
> 
> The pathways are broken without this change. I encountered the issue when making the screen captures. You can test it to make sure

The pathway is not broken the issue is known and is link to an issue of the Celo faucet.

**Comment by pyr0gan** [comment]
> > > Rogan do not rename variable of the UI. Thanks you
> > 
> > 
> > The pathways are broken without this change. I encountered the issue when making the screen captures. You can test it to make sure
> 
> The pathway is not broken the issue is known and is link to an issue of the Celo faucet.

Screen Shot 2021-09-07 at 6 32 16 PM

The alert does not appear before the change I made 🤷 . If there is another way to do it I am not seeing, that's OK - but I needed to do this to make the Swap step functional. You can deny the commit if it breaks something.

**Comment by zurgl** [comment]
> > > > Rogan do not rename variable of the UI. Thanks you
> > > 
> > > 
> > > The pathways are broken without this change. I encountered the issue when making the screen captures. You can test it to make sure
> > 
> > 
> > The pathway is not broken the issue is known and is link to an issue of the Celo faucet.
> 
> Screen Shot 2021-09-07 at 6 32 16 PM
> 
> The alert does not appear before the change I made  . If there is another way to do it I am not seeing, that's OK - but I needed to do this to make the Swap step functional. You can deny the commit if it breaks something.

the name of the variable returned by the swap function on `api/celo/swap.ts` is hash not txHash

Screenshot_20210908_024000

Then if you try to do 

```typescript
                const txHash = res.data.txHash
```
it will not work.
Thanks to let me manage this

File: components/protocols/tezos/components/steps/Transfer.tsx [hunk 2]
```diff
@@ -97,7 +97,7 @@ const Transfer = () => {
               <Text strong>Transfer confirmed!</Text>
             }
             description={
-              <a href={transactionUrl(hash ?? '')} target="_blank" rel="noreferrer">
+              <a href={transactionUrl(txHash ?? '')} target="_blank" rel="noreferrer">
```
Comment (zurgl) [inline components/protocols/tezos/components/steps/Transfer.tsx:100]: again this is link to PR#31 if you haven't change anything everything will be ok now.
We're just working again on the broken change of the PR#31.

Comment (zurgl) [reply components/protocols/tezos/components/steps/Transfer.tsx:100]: @pyr0gan In fact this worst than this you haven't updated your repository with a git pull then you're just fixing your own broken change you do before.
celo-ok

**Comment by zurgl** [comment]
```
In Swap.tsx, in the exchangeUSD function, the response from the API route was res.data.hash, which prevents line 50 ({txHash &&) from evaluating to true, so the alert did not show.

In pages/api/celo/swap.ts, the ResponseT was returning the field as hash. Again, this prevented the conditional rendering of the component from evaluating to true.
```

the value `res.data.hash` only live inside the `exchangeUSD` function and do not have any impact outside.
Then this cannot prevents line 50 from evaluating to true as we copy `res.data.hash` to `txHash` using  `setTxHash`

```typescript
    const exchangeUSD = () => {
                setFetching(true)
		axios
			.post(`/api/celo/swap`, state)
			.then(res => {
                                const hash = res.data.hash
                                const celo = res.data.celo
                                setTxHash(hash)
                                setCelo(celo)
			        setFetching(false)
			})
....
```

**Pull request closed without merging**